### PR TITLE
bump rocket-chip to pick up fix for bundle bridge parenthesis

### PR DESCRIPTION
**Pull request by mwachs5**
This bumps soc-testsocket-sifive to pick up fix in rocket-chip for broken backwards compatibility of BundleBridgeSink without parens

### CODE REVIEW

File: wit-manifest.json
Comment (richardxia) [inline wit-manifest.json]: Reminder to update this to the merge commit after the associated PR is merged in.

**Comment by mwachs5** [comment]
yeah meant to add "DO NOT MERGE" but that label didn't exist and I got distracted

**Review by richardxia — APPROVED** [review]
LGTM!

**Pull request merged**
